sawaliram_auth/views.py

```diff
@@ -120,15 +120,39 @@ def post(self, request):
                         'form': form,
                         
                     }
+                    test_mail={'Subject':'Sawaliram - verify your email',
+                         'From':'mail@sawaliram.org',
+                          'To': user.email,
+                          'Message':'Hello ' + user.first_name + ' Thank you for signing up with Sawaliram!'
+                         }
+                    print(test_mail)
```

Comment:
Is this for debugging?

Reply:
It is for debugging, yes, but I don't know why it's included in this PR?